src/fiatExchanges/quotes/normalizeQuotes.ts

```diff
@@ -81,7 +82,7 @@ export function normalizeFiatConnectQuotes(
         ([key, value]: [string, FiatAccountTypeQuoteData | undefined]) => {
           try {
             const normalizedQuote = new FiatConnectQuote({
-              quote,
+              quote: { ...quote, tokenId },
```

Comment:
Instead of adding the tokenId to the `quote` response object we get back from fiat connect response or external quote response, could we just pass it as a separate arg to the FiatConnectQuote or ExternalQuote classes? E.g.,

```
const normalizedQuote = new FiatConnectQuote({
  quote,
  fiatAccountType: key as FiatAccountType,
  flow,
  tokenId
}
```

That way we can avoid all the intermediate `WithTokenId` types